File: Python.cpp
#include <iostream>
#include <stack>
using namespace std;

int main() {
    // Массив фиксированного размера
    int arr[5] = {1, 2, 3, 4, 5};
    cout << "Элементы массива: ";
    for (int i = 0; i < 5; i++)
        cout << arr[i] << " ";
    cout << endl;

    // Стек
    stack<int> st;
    st.push(10);
    st.push(20);
    st.push(30);

    cout << "Верхний элемент стека: " << st.top() << endl;
    st.pop();
    cout << "Верхний элемент: " << st.top() << endl;

    return 0;
}

File: Python.java
import java.util.Arrays;
import java.util.Stack;

public class Python {
    public static void main(String[] args) {
        // Массив фиксированной длины
        int[] arr = {1, 2, 3, 4, 5};
        System.out.println("Элементы массива: " + Arrays.toString(arr));

        // Стек
        Stack<Integer> stack = new Stack<>();
        stack.push(10);
        stack.push(20);
        stack.push(30);

        System.out.println("Верхний элемент стека: " + stack.peek());
        stack.pop();
        System.out.println("Верхний элемент: " + stack.peek());
    }
}